exercises/complex-numbers/description.md

```diff
@@ -1,3 +1,30 @@
 A complex number is a number in the form `a + b * i` where `a` and `b` are real and `i` satisfies `i^2 = -1`.
 
+`a` is called the real part and `b` is called the imaginary part of `z`.
+The conjugate of the number `a + b * i` is the number `a - b * i`.
+The absolute value of a complex number `z = a + b * i` is a real number `|z| = sqrt(a^2 + b^2)`. The square of the absolute value `|z|^2` is the result of multiplication of `z` by its complex conjugate.
+
+The sum / difference of two complex numbers involves adding / subtracting their real and imaginary parts separately:
```

Comment:
Why the spaces between `sum / difference` and `adding / subtracting`? I think I would prefer it without the spaces, to better visually represent that they belong together.

Reply:
I agree. I removed the spaces.